### ISSUE TITLE
Render in dark mode 

### ISSUE THREAD

**Issue by PrinceP** [TeamMeow/vscode-math-to-image#8]
The svg saved doesn't work for dark mode in GitHub pages. Is there a workaround?

![Screenshot 2021-08-31 at 8 48 46 AM](https://user-images.githubusercontent.com/10251537/131436229-c382c094-3b02-408d-8fae-3956dfbf4e6b.png)

**Comment by dzylikecode** [comment]
Yes, I have the same problem. It seems that the background color of the image doesn't work.


- solved

I find a solution. You should modify the link like the following:

```Html
<div align="center"><img src="https://render.githubusercontent.com/render/math?math=\color{red}\bbox[white]{E=mc^2}"></div>
```
it works :

<div align="center"><img src="https://render.githubusercontent.com/render/math?math=\color{red}\bbox[white]{E=mc^2}"></div>

Now, I just add such a feature to the plugin. You can check it at #13.
